api/src/main/java/edu/cornell/mannlib/vitro/webapp/servlet/setup/RDFFilesLoader.java

```diff
@@ -207,5 +214,133 @@ private static String locateHomeDirectory() {
 	private RDFFilesLoader() {
 		// Nothing to initialize.
 	}
+	
+	/**
+	 * Check if the user model (UI) changes conflict with the changes made to
+	 * the firsttime.  If there is conflict, the user model UI value will be
+	 * left unchanged.
+	 * 
+	 * @param baseModel firsttime backup model
+	 * @param userModel current state in the system (user/UI-model)
+	 * @param changesModel the changes between firsttime-files and firsttime-backup
+	 */
+	public static void checkUiChangesOverlapWithFileChanges(Model baseModel,
```

Comment:
If I understood, it excludes some triples from changesModel. 
Maybe it would be better to rename it to something like excludeChangesMadeInUi ?

Reply:
changed to removeChangesThatConflictWithUIEdits()